oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -82,36 +85,49 @@ steps to see if anyone would like to take ownership of it:
    into the `edx organization`_ and follow the `Archive Steps`_ below for the
    forked repo.
 
+
 Archive Steps
 -------------
+
 1. Update the README.rst file in the repository to state that it is archived,
    using the `README Archive Statement`_ below.
 
-2. Add ``archived: True`` to openedx.yaml (creating the file if necessary). Note
-   that it is not necessary for the openedx.yaml file to define an owner when
-   archived is set to True.
+2. Update the openedx.yaml file, creating it if necessary:
+
+   - Add ``archived: True``.
+
+   - Remove the ``openedx-release`` key if it is present.
+
+   - It is not necessary for the openedx.yaml file to define an owner
+     for archived repos.
 
 3. Create an IT help ticket to update the description of the repository to
-   begin with ``[ARCHIVED]`` and for the repository to be archived per `github's
+   begin with ``[ARCHIVED]`` and for the repository to be archived per `GitHub's
    archive process`_
 
-.. _github's archive process: https://help.github.com/en/articles/archiving-repositories
+.. _GitHub's archive process: https://help.github.com/en/articles/archiving-repositories
+
 
 README Archive Statement
 ------------------------
+
 Include this statement in the README.rst file:
 
     This repository has been archived and is no longer supported—use it at your
     own risk. This repository may depend on out-of-date libraries with security
     issues, and security updates will not be provided. Pull requests against
     this repository will also not be merged.
 
+
 For Repos that are Forks
 ------------------------
-If the repository is a fork of an upstream repository that is not within the edX organization, and will no longer be maintained, it can be deleted with the following steps.
 
-1. Review any differences between the upstream parent and the forked repo. Capture anything
-   that should be kept for historical purposes.
+If the repository is a fork of an upstream repository that is not within the
```

Comment:
Did you want to add that steps for moving the fork to the edx-unsupported org?

Reply:
Added. Ready for review.